pkg/rancher-desktop/components/ImagesButtonAdd.vue

```diff
@@ -32,21 +24,15 @@ export default Vue.extend({
 </script>
 
 <style lang="scss" scoped>
-button.btn {
-  display: flex;
-  width: 32px;
-  height: 32px;
-  border-radius: 50%;
-  align-items:center;
-  justify-content:center;
-}
 
-.icon {
-  width: 0.75rem;
-  height: 0.75rem;
-}
+  .btn {
+    margin-top: -3px;
+  }
```

Comment:
What's the benefit to the negative margin? I don't think that we would want this for the button.

Reply:
+1 on Phillip's comment -- I don't see the difference between this code and leaving out the margin-top directive. For those of us who aren't CSS experts, less (CSS code) is more.

Reply:
It was supposed to put the button at the same distance from top and bottom lines of the header:

`margin-top -3px` :
![image](https://user-images.githubusercontent.com/26394656/208622839-8590aed8-069b-448e-bea1-e5ef4f34b83d.png) 
![image](https://user-images.githubusercontent.com/26394656/208623464-7fcb9465-4b33-4a4a-946a-18a27371b01f.png)


`margin-top 0` :
![image](https://user-images.githubusercontent.com/26394656/208623089-f5f19d75-197a-4fc5-b430-e6f43dea274b.png)

Let's remove the margin directive, we can always fix the layout later.